### ISSUE TITLE
Can't see the full filename (see the picture)

### ISSUE THREAD

**Issue by sorawee** [brownplt/code.pyret.org#192]
![screenshot from 2017-07-14 12-23-05](https://user-images.githubusercontent.com/9099577/28225282-57393fae-688f-11e7-8a5f-6409cbb29536.png)

**Comment by blerner** [comment]
This is by design.  When the window is narrow (e.g. ~800-1200px for a projector screen), we wanted to ensure the buttons will all still fit on one row.  So we elided the middle of the name.

**Comment by sorawee** [comment]
So there's no way to see the full name _at all_? I'm fine with the current design (in fact, I like it) as long as there is a way to see the full name.

**Comment by blerner** [comment]
Currently no.  We probably could add a tooltip that appears on hover.  @jpolitz whadayathink?

**Comment by jpolitz** [comment]
Yeah, good point.  The only way to see the full name is to use rename or save as right now.  A hover would help a lot.

**Comment by YanyanR** [comment]
Do we want something like this? 
https://www.w3schools.com/css/tryit.asp?filename=trycss_tooltip

**Comment by schanzer** [comment]
I've opened a PR for this in https://github.com/brownplt/code.pyret.org/pull/571
